editor/components/webgl/surface.ts

```diff
@@ -113,9 +113,7 @@ export class WebGLSurface implements AfterViewInit, OnDestroy {
     }
 
     ngOnDestroy(): void {
-        this._loop = () => {
-            this.gl = undefined;
-        };
+        this._loop = () => {};
```

Comment:
Yes! It's a way to stop the `requestAnimationFrame`. To be honest I think, this code should instead store the id returned by `requestAnimationFrame` and use `clearAnimationFrame` which is way cleaner.